### DISCUSSION TITLE
pinGitHubActionDigests not working for `oracle-actions/setup-java@v1`

### DISCUSSION THREAD

**Discussion by marcphilipp** [renovatebot/renovate#31342] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

In https://github.com/junit-team/junit5/pull/3958, most GitHub Actions were pinned to their digest, but `oracle-actions/setup-java@v1` was not. It's still being referenced without digest:
https://github.com/junit-team/junit5/blob/a82af6b91e2ffc7271b47c705bc057031ea5fb11/.github/workflows/cross-version.yml#L38

Why is that and what do I need to do so it can get pinned?

**Comment by rarkins** [comment, marked as answer]
Here's the root cause/solution required: https://github.com/renovatebot/renovate/issues/28016

`v1` in this repo is a branch, and Renovate is using the github API to query tags only
